livekit-agents/livekit/agents/llm/function_context.py

```diff
@@ -105,7 +105,7 @@ class CalledFunction:
 def ai_callable(
     *,
     name: str | None = None,
-    description: str | _UseDocMarker | None = None,
+    description: str | _UseDocMarker | None = USE_DOCSTRING,
```

Comment:
we don't need the `None` value anymore? it seems that we need either a description, or USE_DOCSTRING